references/cicd-sharedflow-pipeline/.eslintrc-jsc.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
```

Comment:
If we want to demonstrate triggering the linter from maven, please could we add some javascript to the shared flow to demonstrate this?

Reply:
@laughingbiscuit - Was there more for a structure so that when any one clones this repo and then extends this to add JS policies, they have these up front so that it is not missed. 

If you think this repo is not meant for that, then I can remove those references as I dont intend to add any JS to the sharedflow. Please advice

Reply:
to keep it simple, lets remove.

Reply:
done